cirq/contrib/quimb/mps_simulator.py

```diff
@@ -306,10 +318,14 @@ def __init__(
             sum_prob_atol: Because the computation is approximate, the sum of
                 the probabilities is not 1.0. This parameter is the absolute
                 deviation from 1.0 that is allowed.
+            grouping: How to group qubits together, if None all are individual.
             initial_state: An integer representing the initial state.
         """
         self.qubit_map = qubit_map
+        self.grouping = qubit_map if grouping is None else grouping
```

Comment:
Should we verify that all `qubit_map` qubits appear in `grouping`?

Reply:
Done. Also added a test.